R/FilterState.R

```diff
@@ -76,6 +76,7 @@ FilterState <- R6::R6Class( # nolint
                           varlabel = character(0),
                           dataname = NULL,
                           extract_type = character(0)) {
+      checkmate::assert_class(x_reactive, "function")
```

Comment:
lets change the name as it's not reactive anymore - especially as there's private$x_reactive which is reactive inside the class - it's a shame we need both the function and the reactive of the evaluated function - would be nice not to, though suspect we do need them both

Reply:
Simplified initialize so it doesn't have two places where code is evaluated. The only thing to discuss is a naming